modules/nlu/src/backend/intents/intent-featurizer.ts

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash'
+
+import Utterance from '../utterance/utterance'
+
+function getEntitiesEncoding(utt: Utterance, customEntities: string[]): number[] {
```

Comment:
This function looks like its copied from `context-featurizer.ts`. You should choose only one emplacement.